tools/flash-shelly.py

```diff
@@ -372,6 +381,47 @@ def is_newer(v1, v2):
   else:
     return False
 
+def write_static_ip(device_info, network_type, ipv4_ip='', ipv4_mask='', ipv4_gw='', ipv4_dns=''):
+  wifi_ip = device_info.wifi_ip
+  if device_info.fw_type == 'homekit':
+    if network_type == 'static':
+      message = f"Configuring static IP to {ipv4_ip}..."
+      mask_value={'config': {'wifi': {'sta': {'netmask': ipv4_mask}}}}
+      dns_value={'config': {'wifi': {'sta': {'nameserver': ipv4_dns}}}}
+      gw_value={'config': {'wifi': {'sta': {'gw': ipv4_gw}}}}
+      ip_value={'config': {'wifi': {'sta': {'ip': ipv4_ip}}}}
```

Comment:
Why not this way?
```suggestion
      value={'config': {'wifi': {'sta': {'netmask': ipv4_mask, 'nameserver': ipv4_dns, 'gw': ipv4_gw, 'ip': ipv4_ip}}}}
```
and then one single request to save it?

Reply:
yeah, that is a lot cleaner.